app/uk/gov/nationalarchives/omega/editorial/models/EditSet.scala

```diff
@@ -45,18 +45,19 @@ case class EditSetRecord(
   background: String,
   custodialHistory: String,
   relatedMaterial: Seq[RelatedMaterial] = Seq.empty,
-  separatedMaterial: Seq[SeparatedMaterial] = Seq.empty
+  separatedMaterial: Seq[SeparatedMaterial] = Seq.empty,
+  creatorIDs: Seq[String] = Seq.empty
```

Comment:
The `EditSetRecord` should have a sequence of "creators" not "creatorIDs".

Reply:
👀

Reply:
I would have thought that the creator IDs were just pointers to reference data, similar to `legalStatus` and `placeOfDeposit`; the reference data in the case is the combined list of corporate bodies and persons.

Even if these came over the wire, the application would ignore all fields except the `id`.

Reply:
If we want to make that distinction @tna-erasmos then we should rename `legalStatus` and `placeOfDeposit` to be consistent.

Reply:
So, in `EditSetRecord`, you'd like for me to rename `legalStatus` to `legalStatusID` and `placeOfDeposit` to `placeOfDepositID`?

Reply:
I'll assume so and make the change.

Reply:
Changed, @rwalpole.

Reply:
Thanks @tna-erasmos